internal/cli/feedback/stdio.go

```diff
@@ -16,17 +16,37 @@
 package feedback
 
 import (
+	"errors"
 	"io"
 )
 
-// OutputStreams returns the underlying io.Writer to directly stream to
+// DirectStreams returns the underlying io.Writer to directly stream to
 // stdout and stderr.
-// If the selected output format is not Text, the returned writers will
-// accumulate the output until command execution is completed.
+// If the selected output format is not Text, the function will error.
+// Using this function will allow usage of the PrintResult anymore.
```

Comment:
> I wasn't able to understand the meaning of this sentence.

Because it doesn't make any sense :-)

I wanted to say that once you get the streams from `DirectStream` you must not use the `PrintResult` anymore because you are taking direct control of the output streams.

So maybe I can rephrase it like:

> Using the streams returned by this function allows direct control of the output and the PrintResult function must not be used anymore.

Reply:
The rephrased wording looks great.